yellowbrick/text/dispersion.py

```diff
@@ -58,19 +71,30 @@ class DispersionPlot(TextVisualizer):
     process, but can and should be set as early as possible.
     """
 
-    def __init__(self, words, ax=None, color=None, ignore_case=False,
-                 annotate_docs=False, **kwargs):
+    # NOTE: cannot be np.nan
+    NULL_CLASS = None
+
+    def __init__(self, words, ax=None, colors=None, ignore_case=False,
```

Comment:
Is it necessary for `words` to be passed in on instantiation? It looks like the user has to pass `text` when they call `fit()`, and that `words` is either the same as `text` or could be derived from it.

Reply:
@rebeccabilbro I am not sure that I understand.   `words` is the list of targeted words that the user wants to be marked throughout the document.  These are the words that appear on the y-axis.

Reply:
Gotcha! I think I'm getting mixed up because we have both `self.words` and `self.target_words_` here. To make it a bit easier to understand, could we perhaps change `self.words` to `self.target_words` and `self.target_words_` to `self.indexed_words_`, or something like that?